prometheus/histogram.go

```diff
@@ -732,6 +750,8 @@ type histogram struct {
 
 	// afterFunc is for testing purposes, by default it's time.AfterFunc.
 	afterFunc func(time.Duration, func()) *time.Timer
+
+	nativeExemplars nativeExemplars
```

Comment:
Nit: Maybe move this up to just after `resetScheduled` as `now` and `afterFunc` are just for testing purposes, while `nativeExemplars` is important for regular use.